tests/unit_tests/data_steward/retraction/create_deid_map_test.py

```diff
@@ -0,0 +1,125 @@
+# Python imports
+import unittest
+
+# Third party imports
+import mock
+import pandas as pd
+
+# Project imports
+import retraction.create_deid_map as create_deid_map
+import utils.bq
+
+
+class CreateDeidMapTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = 'test_project_id'
+        self.dataset_id = 'test_dataset_id'
+        self.all_datasets = [
+            'R2021Q1R1_combined', 'R2021Q1R1_deid', 'R2021Q1R2_combined',
+            '2021q1r3_combined', 'R2021Q1R3_deid'
+        ]
+        self.deid_datasets = ['R2021Q1R1_deid', 'R2021Q1R3_deid']
+        self.combined_datasets = [
+            'R2021Q1R1_combined', 'R2021Q1R2_combined', '2021q1r3_combined'
+        ]
+
+    @mock.patch('retraction.create_deid_map.utils.bq.list_datasets')
+    def test_get_combined_datasets_for_deid_map(self, mock_list_datasets):
+        result = create_deid_map.get_combined_datasets_for_deid_map(
+            self.project_id)
+
+        expected_all_datasets = mock_list_datasets.return_value = utils.bq.list_datasets(
+            self.project_id)
+        expected_deid_datasets = self.deid_datasets
```

Comment:
It would be good to see more values here. I cannot see the combined datasets being generated. I would suggest all_datasets to be combined, deid and other types of datasets, and the assert checks if combined datasets are output as expected. 
From my other suggestion, it would also help to see a mapping so we can easily tell which deid datasets have a combined and which don't. The test case could have unmapped deid and combined datasets.